### ISSUE TITLE
PeriodicJob

### ISSUE THREAD

**Issue by avasapollo** [taylorchu/work#19]
Hey guys do we still support the periodic jobs?
I managed to use the schedule job at runtime, but I did not able to set up a periodic job

**Comment by taylorchu** [comment]
yes. but we simply use https://godoc.org/github.com/robfig/cron with enqueuer. as a result, it does not have to be in gocraft/work.

**Comment by avasapollo** [comment]
I wondering how we suppose to handle the cron logic with multiple pods on Kubernetes.
it should be synchronized the work v1 handle that situation.

**Comment by taylorchu** [comment]
could you explain what that cron logic is with multiple pods?

**Comment by taylorchu** [comment]
if you don't have a single pod that is in charge of adding cron jobs, the unique middleware https://godoc.org/github.com/taylorchu/work/middleware/unique is what you need to specify a dedup key (from queue's namespace and queue id) and a duration.

**Comment by avasapollo** [comment]
yeah sorry I try to clarify better, so the scenario is this, 3 pods of an application, I want a job for every 30 minutes that do something.
the job is picked random from one of that 3 pods and do that job operation.
In work version 1 I need just to create the periodic job and everything is done.
is there this functionality?
Or do you suggest in someway to pick one of that and it will be in charge to schedule a job every 30 seconds? (because like this I need to create a logic that elect one of the pod and pay attention if that pod is still alive)

**Comment by taylorchu** [comment]
so that `unique` package above is what you need. after this middleware is used on enqueue, although all 3 pods will all try to enqueue, it will be deduped (only one pod will actually enqueue successfully).

**Comment by avasapollo** [comment]
ok I got the point that just one pod will process the action, but  who is going to enqueue  the unique job every 30 seconds? Because if I use just cron to enqueue the job all the 3 pods will equeue the job

**Comment by nickdelja** [comment]
It sounds like all three pods are going to try to enqueue, but only one will succeed.  Can this issue be closed?
